Sources/CustomerCenter/CustomerCenterPresentationMode.swift

```diff
@@ -18,10 +18,10 @@ import Foundation
 /// Presentation options to use with the [presentCustomerCenter](x-source-tag://presentCustomerCenter) View modifiers.
 public enum CustomerCenterPresentationMode {
 
-    /// Customer center presented using SwiftUI's `.sheet`.
+    /// Customer Center presented using SwiftUI's `.sheet`.
```

Comment:
The double backticks are used to link to a specific symbol/function. I think we'd need to use the full reference to a specific .sheet function. Since there are multiple .sheet functions, I think it's best to keep it as is and not mention a specific implementation in the docs in case we change it in the future